xml/security_ldap.xml

```diff
@@ -405,11 +409,80 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
 
   <sect2 xml:id="sec.security.ldap.server.setup">
   <title>Installation and Basic Setup</title>
-   <para>Install the 389-ds package:</para>
+   <para>
+    The <systemitem>389-ds</systemitem> package contains the &ds398; and the
+    administration tools. If the package is not installed yet, install it with
+    the following command:</para>
   <screen>&prompt.sudo;zypper install 389-ds</screen>
- <!--next steps:
-     create instance.inf
-     dscreate from-file etc.-->
+ <procedure>
+  <title>Setting up the &ds389; Instance</title>
+    <para>
+     You create the instance with the <command>dscreate</command> command.
+     It can take a configuration file (<filename>*.inf</filename>) which
+     defines the instance configuration settings. Alternatively, it can be run
+     in an interactive mode. To create a template <filename>*.inf</filename> file,
+     use the <option>create-template</option> option. For more details, see the
+     man page of <command>dscreate</command>.
+    </para>
+    <para>
+     If you want to set up a trial instance, you can use the following
+     example <filename>*.inf</filename> file as a starting point:
```

Comment:
I'm worred about adding in a screenshot/file as we may change the content in the future ... but maybe that's a problem if we "get there".

Reply:
Nothing is carved in stone and I'm aware that updates might be required every now and then. (That's also why I chose to go with the 'basic' example from the upstream docs, because it does not contain too many details).